spec/controllers/ops_controller/ops_rbac_spec.rb

```diff
@@ -264,9 +265,7 @@
         session[:edit] = edit
       end
 
-      after(:each) do
-        expect(response.status).to eq(200)
-      end
+      after { expect(response.status).to eq(200) }
```

Comment:
You shouldn't test in an after hook if that's possible, duplication here is allowed.